apps/web/components/dialogs/session/session-button.tsx

```
@@ -63,22 +62,16 @@ export function SessionButton() {
               <li
                 onClick={openConnectModal ? openConnectModal : openAccountModal}
-                className="w-full cursor-pointer text-center flex justify-center gap-x-1"
+                className="hover:text-accent-400 active:text-accent-400 w-full cursor-pointer text-center flex justify-center gap-x-1"
               >
                 <Wallet />
                 &nbsp;{' '}
-                <span className="hover:text-accent-400 active:text-accent-400">{account?.address ? formatAddressShort(account.address) : 'Connect'}</span>
+                <span>{account?.address ? formatAddressShort(account.address) : 'Connect'}</span>
               </li>
 
               <li onClick={logout} className="hover:text-accent-400 active:text-accent-400 cursor-pointer flex justify-center items-center gap-x-3 pb-5 border-b w-full border-b-textInfoForeground/50">
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)